project 1 html-css/Index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="style.css" rel="stylesheet"/>
+</head>
+
+
+<body>
+    <!-- header bar -->
+    <header class="top-bar">
+        <section class="company-name">My Logo</section>
+        <nav class="nav-btn">
+            <img  src="Enabled.png" alt="">
+        </nav>
+    </header>
+
+    <section class="main-pg">
+
+        <div class="hey">
+        Hey There.
+        <br>
+        I'm Ahmad.
+        </div>
```

Comment:
Consider using `<h1>`  for main headings (“Hey There.” and “I’m Ahmad.”), which helps with accessibility and SEO.